.github/workflows/update-changelog.yml

```
@@ -0,0 +1,124 @@
+              }]
+            }' \
+            "${{ secrets.DISCORD_WEBHOOK_URL }}"
+      
+      - name: Add success reaction to comment
+        if: success()
+        run: |
+          curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            -X POST \
+            "https://api.github.com/repos/${{ github.repository }}/issues/comments/${{ github.event.comment.id }}/reactions" \
+            -d '{"content":"rocket"}'
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)